docs/client_configuration.rst

```diff
@@ -0,0 +1,79 @@
+Client Configuration
+====================
+
+`elasticsearch-lua`_ was designed with the aim to allow users to configure
+almost all of the parameters. The standard way of creating and configuring
```

Comment:
elasticsearch-lua was designed to allow users to configure almost all parameters.